packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md

```diff
@@ -1,28 +1,25 @@
-Display the user info by updating the `showUserInfo` function of `ContentView`:
+Display the user info by updating the `showUserInfo()` function of `ContentView.swift`:
 
 ```swift
 func showUserInfo() {
-  guard let authStateManager = authStateManager else {
-    showError(title: "Unable to Show User Info",
-              message: "No Okta client found.")
-
-    return
-  }
-
-  authStateManager.getUser() { response, error in
-    if let error = error {
-      showError(title: "Unable to Read User Info", error: error)
+   guard let credential = Credential.default,
+         let idToken = credential.token.idToken else {
+      showError(title: "Unable to Show User Info",
+                message: "Could not read the token for the current user.")
       return
-    }
+   }
 
-    if response != nil {
-      let sortedResponse = response!.sorted { return $0.key < $1.key }
-      var userInfoText = ""
-      sortedResponse.forEach { userInfoText += ("\($0): \($1) \n") }
-      infoText = userInfoText
-    }
-  }
+   var infoString = ("Name: \(idToken.name!)\n")
+   infoString += "User Name: \(idToken.preferredUsername!)\n"
+   infoString += "User ID: \(idToken.subject!)\n"
+   if let issueDate = idToken.issuedAt {
+      let dateFormatter = DateFormatter()
+      dateFormatter.timeStyle = .medium
+      dateFormatter.dateStyle = .medium
+      infoString += "Issue Date: \(dateFormatter.string(from: issueDate))\n"
+   }
+   infoText = infoString
 }
 ```
 
-The completion handler in `getUser` is called when the Okta org server returns a result. Your production app may present a busy cursor and status while it waits for the callback. The completion handler then shows an error if appropriate or sorts the user info keys before displaying the raw data.
\ No newline at end of file
+The function reads the user information from the local ID token. The available information depends on the scopes that are specified in the `Okta.plist` file. You can also request information from the server with the `userInfo()` of the `Credential` class.
```

Comment:
It might be nice to clarify what information is being requested with the userInfo call.

```suggestion
The function reads the user information from the local ID token. The available information depends on the scopes that are specified in the `Okta.plist` file. You can also request the full user's profile from the server with the `userInfo()` of the `Credential` class.
```

Reply:
AFAIK the issue is that it returns only the claims that are consistent with the scopes. "full user profile" isn't actually descriptive because it can mean many things. The only way to give a somewhat accurate description of what's returned is to talk about the underlying endpoints called by `userInfo()`, but that assumes those won't change.

Thoughts?

Reply:
Hmm, how about simply "You can also request additional information about the user from the server with the ....."

Reply:
How about something slightly shorter:

> You can also request user information from the server...